Adyen/httpclient.py

```diff
@@ -35,10 +35,9 @@
 
 
 class HTTPClient(object):
-    def __init__(self, app_name, user_agent_suffix,
-                 lib_version, force_request=None):
+    def __init__(self, user_agent_suffix, lib_version, force_request=None):
         # Check if requests already available, default to urllib
-        self.user_agent = app_name + " " + user_agent_suffix + lib_version
+        self.user_agent = " " + user_agent_suffix + lib_version
```

Comment:
I don't think we need the empty space at the beginning
```suggestion
        self.user_agent = user_agent_suffix + lib_version
```